shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RealtimePatterns.kt

```diff
@@ -63,7 +63,7 @@ sealed class RealtimePatterns {
         val route: Route,
         val headsign: String,
         val line: Line?,
-        override val patterns: List<RoutePattern>,
+        override val patterns: List<RoutePattern?>,
```

Comment:
question: When would this be null?

Reply:
If there's an added trip with no route pattern that gets sorted into this `RealtimePatterns`, then `patterns` will contain `null`.

Reply:
Ah, right! May be worth a comment about that edge case

Reply:
Commented.